docs/src/intro.md

```diff
@@ -96,6 +96,7 @@ Implemented features include:
 - Export of reports to PDF, both via the UI as well as via the API.
 - Notifications of events, such as metrics turning red, to Microsoft Teams.
 - Side-by-side comparison of measurements at different points in time.
+- Integration with issue tracker (Jira only at the moment) to manage actions and technical debt.
```

Comment:
Azure DevOps issues/work items are also integrated

Reply:
Unfortunately not, only Jira can be configured as issue tracker.